app/config/ezplatform.yml

```diff
@@ -37,5 +37,13 @@ ezpublish:
                 full:
                     home:
                         template: full\home.html.twig
+                        controller: AppBundle:Site:show
                         match:
                             Id\Location: 2
+
+            location_view:
+                row:
```

Comment:
We can discuss the naming. I am referring to twitter bootstrap.
But line is fine :)